src/hotspot/share/opto/idealGraphPrinter.cpp

```diff
@@ -676,7 +672,11 @@ void IdealGraphPrinter::visit_node(Node* n, bool edges) {
       } else if (t->base() == Type::Return_Address) {
         print_prop(short_name, "RA");
       } else if (t->base() == Type::AnyPtr) {
-        print_prop(short_name, "P");
+        if (t->is_ptr()->ptr() == TypePtr::Null) {
+          print_prop(short_name, "NULL");
```

Comment:
I'm surprised this doesn't trip over sources/TestNoNULL.java.

Reply:
Actually it does, see failures in github actions testing:
> Failed. Execution failed: `main' threw exception: java.lang.RuntimeException: Test found 1 usages of 'NULL' in source files. See errors above.

Reply:
I don't think this should be given an exclusion in TestNoNULL.

If the printed name is supposed to be related to the compiler object `TypePtr::Null` then I suggest it should
be "Null".  If it's supposed to be related to the Java null value, then "null" seems appropriate.

Reply:
I guess `Null` is fine then. I only chose `NULL` to be in line with the other capitalized letters shown in IGV in the condensed view.